jackal_control/config/localization.yaml

```diff
@@ -18,7 +18,7 @@ ekf_node:
     odom0_differential: false
     odom0_queue_size: 10
 
-    imu0: imu/data
+    imu0: sensors/imu_0/data_raw
```

Comment:
This should not be changed. Add `<frameId>` in the Gazebo `<sensor>`. 
See https://github.com/gazebosim/gz-sensors/pull/444

Depending on the Gazebo version it could also be `<gz_frame_id>`, `<ign_frame_id>` or `<ignition_frame_id>`